### PR TITLE
add screening, movie and cinema hall models

### PR DESCRIPTION
**Pull request by atytko**
Added screening, movie and cinema hall models.

### CODE REVIEW

**Review by baniak66 — APPROVED** [review]
LGTM 👍

File: app/models/cinema_hall.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class CinemaHall < ApplicationRecord
```
Comment (kodowicz) [inline app/models/cinema_hall.rb:1]: these files should have `# frozen_string_literal: true`. Have you run rubocop? You can install extension in vsc to see robocop's prompts immediately:)

File: db/migrate/20210806104723_create_movies.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateMovies < ActiveRecord::Migration[6.1]
+  def change
+    create_table :movies do |t|
+      t.string :title, null: false
+      t.text :description, null: false
+      t.interval :duration, null: false
```
Comment (kodowicz) [inline db/migrate/20210806104723_create_movies.rb:6]: I think the best type would be just `Time` 🤔

Comment (Sonickers) [reply db/migrate/20210806104723_create_movies.rb:6]: Or even `integer` and store minutes :)

Comment (baniak66) [reply db/migrate/20210806104723_create_movies.rb:6]: I never used interval and would go with integer (store number of minutes), but interval should work here as well. https://www.bigbinary.com/blog/rails-6-1-adds-postgresql-interval-data-type

Comment (wmaciejak) [reply db/migrate/20210806104723_create_movies.rb:6]: > **Where there are 10 developers, there will be 11 valid solutions**

I think that all the solutions are valid, so from my perspective, it's a more personal choice ;)

**Review by Sonickers — APPROVED** [review]
👌

File: app/models/cinema_hall.rb (hunk 1 above)
Comment (atytko) [reply app/models/cinema_hall.rb:3]: thanks for spotting Ania! I forgot to run it of course :D

File: db/migrate/20210806104723_create_movies.rb (hunk 2 above)
Comment (atytko) [reply db/migrate/20210806104723_create_movies.rb]: Just corrected it to integer, thank you all :)

**Review by wmaciejak — APPROVED** [review]
Good job, I think that you can move forward 🚀

**Pull request merged**